wagtailnhsukfrontend/static/wagtailnhsukfrontend/css/fixes.css

```diff
@@ -4,3 +4,19 @@
 div.rich-text :last-child {
   margin-bottom: 0;
 }
+
```

Comment:
This CSS is now in the NHS.UK frontend library, is it possible to use the latest version (2.3.0) in this change?

Reply:
Will add a commit updating the version
The fix grey panel commit must have been from some work I was looking at previously in this repo, it still needs to be put in but i will add it as a seperate pull request